<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grov::Main_window</class>
 <widget class="QMainWindow" name="grov::Main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Google Reader Offline Viewer</string>
  </property>
  <widget class="QWidget" name="parent_widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="grov::Viewer" name="viewer" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>549</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="go_offline_action"/>
    <addaction name="flush_offline_data_action"/>
    <addaction name="quit_action"/>
   </widget>
   <widget class="QMenu" name="menu_Feed">
    <property name="title">
     <string>&amp;Feed</string>
    </property>
    <addaction name="previous_item_action"/>
    <addaction name="next_item_action"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Feed"/>
  </widget>
  <action name="quit_action">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="go_offline_action">
   <property name="text">
    <string>Go &amp;offline</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="flush_offline_data_action">
   <property name="text">
    <string>&amp;Flush offline data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="next_item_action">
   <property name="text">
    <string>&amp;Next item</string>
   </property>
   <property name="shortcut">
    <string>J</string>
   </property>
  </action>
  <action name="previous_item_action">
   <property name="text">
    <string>&amp;Previous item</string>
   </property>
   <property name="shortcut">
    <string>K</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>grov::Viewer</class>
   <extends>QWidget</extends>
   <header>viewer.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
